tfx/proto/example_gen.proto

```diff
@@ -111,7 +111,10 @@ enum PayloadFormat {
   // Serialized any protocol buffer.
   FORMAT_PROTO = 11;
 
-  reserved 1 to 5, 8 to 10, 12 to max;
+  // Serialized parquet messages.
+  FORMAT_PARQUET = 12;
```

Comment:
File format is also needed?

e.g., FORMAT_TF_EXAMPLE (payload format) is stored in FORMAT_TFRECORDS_GZIP (file format)

for Parquet, what's the file format?

Reply:
I guess PARQUET is also the file format?

Reply:
@1025KB these are the file formats supported by pyarrow https://arrow.apache.org/docs/python/parquet.html#compression-encoding-and-file-compatibility. Should I add one for each? or should we agree on a set of ones we are planning to support?

Reply:
the default is SNAPPY

Reply:
Not necessary, one splittable file format (parquet) in addition to tfrecord (non-splittable file format) should be enough for now

Reply:
Note that we will use beam.io.writeToParquet to generate output files, so for reading, only need to support the format used by beam.io.writeToParquet

Reply:
While we're in this code, to maximize compatibility I think it would make sense to include gzip, brotli, and none, as well as snappy - unless that creates headaches, in which case snappy and none (and maybe gzip?) would seem sufficient.

Reply:
I think supporting only the format that beam source and sink support should be sufficient for now, other formats could be considered in a follow up if needed